mkt/webapps/tasks.py

```diff
@@ -1120,7 +1112,9 @@ def generate_app_package(app, out, permissions, version='1.0'):
         version=version, manifest=json.dumps(manifest))
 
 
-def generate_packaged_app(name, category, permissions=(), num_versions=1):
+
+def generate_packaged_app(name, apptype, category, permissions=(),
+                          versions=(4,), num_locales=2, **kw):
```

Comment:
Maybe `num_versions`? For a moment there i wasn't sure if versions was a magic constant.